packages/storefront-events-collector/src/utils/aep/productListItems.ts

```diff
@@ -27,9 +31,9 @@ const createProductListItems = (
                 SKU: item.product?.sku,
                 name: item.product?.name,
                 quantity: item.quantity,
-                priceTotal: (item.prices?.price?.value * item.quantity) || 0,
+                priceTotal: item.prices?.price?.value * item.quantity || 0,
```

Comment:
For the sake of consistency with line 36
```suggestion
                priceTotal: item.prices?.price.value * item.quantity || 0,
```

Reply:
i'd rather not change it because i'm not exactly the type we have defined in our code is correct. the type declares that value is always there, but.... i'm not so sure that's the case?

Reply:
Then I'm confused; you said `prices.price` was defined in in our shopping cart type. Either `item.prices.price` should **not** be nullable here, it it **should** be nullable on line 36

Reply:
You are right, it should or shouldn't be but I'm not actually sure these types align with all of the different ways those prices are coming across from magento. But for consistencies sake, i've added the optional chain to the currency on line 36 since that could truly be undefined in the price. Line 34 was only changed because of formatting.